docs/cms/validators.adoc

```diff
@@ -0,0 +1,83 @@
+= Content Validators
+:toc: right
+:imagesdir: ../images
+
+image:xp-780.svg[XP 7.8.0,opts=inline]
+
+== Introduction
+
+XP by default validates nearly created and modified content:
```

Comment:
What is "_nearly created_"? "_nearly **all** created_"?

Reply:
```suggestion
XP by default validates newly created and modified content:
```